helm_deploy/hmpps-change-someones-cell/values.yaml

```diff
@@ -48,9 +48,9 @@ generic-service:
       SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
       SYSTEM_CLIENT_SECRET: "SYSTEM_CLIENT_SECRET"
       SESSION_SECRET: "SESSION_SECRET"
-    elasticache-redis:
-      REDIS_HOST: "primary_endpoint_address"
-      REDIS_AUTH_TOKEN: "auth_token"
+    csc-redis:
```

Comment:
I'm not sure about this. 
In other projects (CVL, WPIP), it is as per elasticache-redis

Reply:
That didn't work. This fixed it.

Reply:
Terraform put the creds in a secret called "csc-redis".

Reply:
ok